Dockerfile

```diff
@@ -54,21 +57,26 @@ ARG ldflags
 # Do not force rebuild of up-to-date packages (do not use -a) and use the compiler cache folder
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} \
     go build -trimpath -ldflags "${ldflags} -extldflags '-static'" \
-    -o manager ${package}
+    -o manager ${package} \
+    && groupadd -r ocne \
+    && useradd --no-log-init -r -m -d /ocne -g ocne -u 1000 ocne \
+    && mkdir -p /home/ocne \
+    && chown -R 1000:ocne /workspace/manager /home/ocne \
+    && chmod 500 /workspace/manager
 
 # Production image
-FROM ghcr.io/oracle/oraclelinux:8-slim
-RUN microdnf update \
-    && microdnf clean all
+
+FROM $FINAL_IMAGE
 WORKDIR /
 COPY --from=builder /workspace/manager .
+
+# Copy users and groups
+COPY --from=builder /etc/passwd /etc/passwd
+COPY --from=builder /etc/group /etc/group
+COPY --from=builder /home/ocne /home/ocne
```

Comment:
Does this need owner and group set explicitly when copying, or does it pick up those from the source directories?